apps/web/pages/settings/organizations/[id]/add-teams.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from "next/router";
+
+import { AddNewTeamsForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout, Meta } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const AddNewTeamsPage = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  if (!router.isReady) return null;
+  return (
+    <>
+      <Meta title={t("create_your_teams")} description={t("create_your_teams_description")} />
+      <AddNewTeamsForm />
+    </>
+  );
+};
+
+AddNewTeamsPage.getLayout = (page: React.ReactElement) => (
+  <>
+    <WizardLayout
+      currentStep={5}
+      maxSteps={5}
+      isOptionalCallback={() => {
+        window.location.replace(`/getting-started`);
+      }}>
```

Comment:
Can we not use router here?

Reply:
Didn’t know it was available from getLayout, done!